packages/features/insights/server/events.ts

```diff
@@ -86,6 +86,17 @@ class EventsInsights {
     return baseBookings;
   };
 
+  static getTotalCompletedEvents = async (bookingIds: number[]) => {
+    return await prisma.bookingTimeStatus.count({
+      where: {
+        id: {
+          in: bookingIds,
+        },
+        timeStatus: "completed",
+      },
+    });
+  };
```

Comment:
This functions it's not related to this problem but found out some irregularities when counting.